src/logging.cc

```diff
@@ -827,6 +833,91 @@ void LogDestination::DeleteLogDestinations() {
   sinks_ = NULL;
 }
 
+namespace {
+
+vector<string> SplitString(const string& s, const char delimiter) {
+  std::stringstream ss(s);
+  string t;
+  vector<string> tokens;
+
+  while (std::getline(ss, t, delimiter)) {
+    if (t.length() > 0) {
+      tokens.push_back(t);
+    }
+  }
+  return tokens;
+}
+
+bool IsGlogLog(const string& log_name) {
+  static const int kGlogFilenameTokenCount = 6;
+  vector<string> log_name_tokens = SplitString(log_name, '.');
```

Comment:
This is broken on windows, e.g., `windoze.exe`.
Splitting filename by `.` is not reliable.

Reply:
Resolved in 77a1f39 by matching the filename keyword by keyword, please see the commit message for details.